tools/verifyio/verifyio.py

```diff
@@ -90,6 +114,17 @@ def verify_pair_proper_synchronization(n1, n2, vio):
 
     # Algorithm 4: On-the-fly MPI check
     if vio.algorithm == 4:
```

Comment:
for commit/posix semantics, next_sync_index is null.

Reply:
fixed